flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -605,17 +605,32 @@ private void callSource(SqlCommandCall cmdCall) {
 
 	private void callCreateDatabase(SqlCommandCall cmdCall) {
 		final String createDatabaseStmt = cmdCall.operands[0];
-		executor.executeUpdate(sessionId, createDatabaseStmt);
+		try {
+			executor.executeUpdate(sessionId, createDatabaseStmt);
+			printInfo(CliStrings.MESSAGE_DATABASE_CREATED);
+		} catch (SqlExecutionException e) {
```

Comment:
call `terminal.flush` here?

Reply:
We don't need this method for `printInfo ` has called `terminal.flush`